README.md

```diff
@@ -366,13 +366,13 @@ Built-in functions. Ready to be bound to any key you like. :smile:
 
 ### File Pickers
 
-| Functions                           | Description                                                                                 |
-|-------------------------------------|---------------------------------------------------------------------------------------------|
-| `builtin.find_files`                | Lists Files in current directory.                                                           |
-| `builtin.git_files`                 | Lists Git files in current directory.                                                       |
-| `builtin.grep_string`               | Searches for a string under the cursor in current directory.                                |
-| `builtin.live_grep`                 | Searches in current directory files. (respecting .gitignore)                                |
-| `builtin.file_browser`              | Ivy-like file explorer. Creates files by typing in filename and pressing `<C-e>`. Press `<C-e>` without prompt for more info |
+| Functions                           | Description                                                                                                                                               |
+|-------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `builtin.find_files`                | Lists files in your current working directory                                                                                                             |
+| `builtin.git_files`                 | Lists files not ignored by git in your current directory                                                                                                  |
+| `builtin.grep_string`               | Searches for the string under your cursor in your current working directory                                                                               |
+| `builtin.live_grep`                 | Search for a string in your current working directory and get results live as you type (respecting .gitignore)                                            |
+| `builtin.file_browser`              | Ivy-like file explorer. Lists files and folders in your current working directory, open files, navigate your filesystem, and create new files and folders | 
```

Comment:
Will do, this is what the previous entry in the README was so I was gonna leave it since someone added it. I think removing it probably better though as it doesn't really describe much to anyone who hasn't used Emacs before.